test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -4597,6 +4604,111 @@ public void testGetGroupsUserMembersOnly() throws Exception {
   }
 
   @Test
+  public void testGetGroups_fullFeeds() throws Exception {
+    insertUsers("User1", "User2", "User3", "User4", "User5");
+    insertGroup("Group1", "User1", "User2", "User3");
+    insertGroup("Group2", "User3", "User4", "User5");
+
+    ImmutableMap<GroupPrincipal, Collection<Principal>> expected =
+        ImmutableMap.of(
+            new GroupPrincipal("Group1", "NS_Local"),
+            ImmutableSet.<Principal>of(
+                new UserPrincipal("User1", "NS"),
+                new UserPrincipal("User2", "NS"),
+                new UserPrincipal("User3", "NS")),
+            new GroupPrincipal("Group2", "NS_Local"),
+            ImmutableSet.<Principal>of(
+                new UserPrincipal("User3", "NS"),
+                new UserPrincipal("User4", "NS"),
+                new UserPrincipal("User5", "NS")),
+            new GroupPrincipal("dm_world", "NS_Local"),
+            ImmutableSet.<Principal>of(
+                new UserPrincipal("User1", "NS"),
+                new UserPrincipal("User2", "NS"),
+                new UserPrincipal("User3", "NS"),
+                new UserPrincipal("User4", "NS"),
+                new UserPrincipal("User5", "NS")));
+    assertEquals(expected, getGroups());
+
+    insertUsers("User6", "User7");
+    insertGroup("Group3", "User6", "User7");
+    // delete group, group members not deleted.
+    deleteGroup("Group2");
+
+    ImmutableMap<GroupPrincipal, Collection<Principal>> expected2 =
+        ImmutableMap.of(
+            new GroupPrincipal("Group1", "NS_Local"),
+            ImmutableSet.<Principal>of(
+                new UserPrincipal("User1", "NS"),
+                new UserPrincipal("User2", "NS"),
+                new UserPrincipal("User3", "NS")),
+            new GroupPrincipal("Group3", "NS_Local"),
+            ImmutableSet.<Principal>of(
+                new UserPrincipal("User6", "NS"),
+                new UserPrincipal("User7", "NS")),
+            new GroupPrincipal("dm_world", "NS_Local"),
+            ImmutableSet.<Principal>of(
+                new UserPrincipal("User1", "NS"),
+                new UserPrincipal("User2", "NS"),
+                new UserPrincipal("User3", "NS"),
+                new UserPrincipal("User4", "NS"),
+                new UserPrincipal("User5", "NS"),
+                new UserPrincipal("User6", "NS"),
+                new UserPrincipal("User7", "NS")));
+    assertEquals(expected2, getGroups());
+  }
+
+  @Test
+  public void testGetGroupsUserMembersOnly_fullAndIncrementalFeeds()
+      throws Exception {
+    insertUsers("User1", "User2", "User3", "User4");
+    insertGroup("Group1", "User1", "User2");
+    insertGroup("Group2", "User3", "User4");
+
+    ImmutableMap<GroupPrincipal, Collection<Principal>> expected =
+        ImmutableMap.of(
+            new GroupPrincipal("Group1", "NS_Local"),
+            ImmutableSet.of(new UserPrincipal("User1", "NS"),
```

Comment:
Wrap these two lines here like the others, with the members all lining up.

Reply:
Looked at these and still missed...:( 
Fixed now :)
Done.